client/ayon_maya/plugins/publish/collect_cache_farm.py

```diff
@@ -0,0 +1,33 @@
+import os
+import pyblish.api
+from ayon_maya.api import plugin
+
+
+class CollectDataforCache(plugin.MayaInstancePlugin):
+    """Collect data for caching to Deadline."""
+
+    # Run after Collect Frames
+    order = pyblish.api.CollectorOrder + 0.4991
+    families = ["publish.farm"]
+    targets = ["local", "remote"]
+    label = "Collect Data for Cache"
+
+    def process(self, instance):
+        # Should we implement this in somewhere else?
+        tmp_staging_dir = os.path.join(os.environ["AYON_WORKDIR"], "cache", "alembic")
+        os.makedirs(tmp_staging_dir, exist_ok=True)
```

Comment:
```suggestion
        tmp_staging_dir = os.path.join(os.environ["AYON_WORKDIR"], "cache", "alembic")
        os.makedirs(tmp_staging_dir, exist_ok=True)
```
I know it is not preferable to do this here(it's still a bit hardcoded. But it would be great if any one can suggest better way to implement the output directory.